pypeit/display/ginga_spec1dview.py

```diff
@@ -0,0 +1,380 @@
+# This is open-source software licensed under a BSD license.
+# Please see the file LICENSE.txt for details.
```

Comment:
Our license file is `LICENSE.rst`, but we (as a project) need to figure out our header and add it to all our source files (which we don't have at the moment).

Reply:
Shall I just remove this comment then? Or change it to `LICENSE.rst`?

Reply:
Removed